operators/dream_texture.py

```diff
@@ -191,7 +195,12 @@ def step_progress_update(self, context):
         bpy.types.Scene.dream_textures_info = bpy.props.StringProperty(name="Info", update=step_progress_update)
         
         info("Waiting For Process")
-        generator = GeneratorProcess.shared()
+        if len(backend_options(self, context)) <= 1:
+            headless_prompt.backend = backend_options(self, context)[0]
```

Comment:
`TypeError: bpy_struct: item.attr = val: expected a string enum, not tuple`
```suggestion
            headless_prompt.backend = backend_options(self, context)[0][0]
```